apps/chat-e2e/src/tests/promptsNames.test.ts

```diff
@@ -0,0 +1,206 @@
+      'EPMRTC-2993',
+      'EPMRTC-2994',
+      'EPMRTC-2997',
+      'EPMRTC-3085',
+    );
+    const prompt = promptData.prepareDefaultPrompt();
+    await dataInjector.createPrompts([prompt]);
+    const newNameWithDot = `${ExpectedConstants.newPromptTitle(1)}.`;
+    const nameWithRestrictedChars = `Prompt${ExpectedConstants.restrictedNameChars}_name`;
+    const expectedPromptName = 'Prompt_name';
+    const longNameWithEmojis =
+      '😂👍🥳 😷 🤧 🤠 🥴😇 😈 ⭐あおㅁㄹñ¿äß😂👍🥳 😷 🤧 🤠 🥴😇 😈 ⭐あおㅁㄹñ¿äß😂👍🥳 😷 🤧 🤠 🥴😇 😈 ⭐あおㅁㄹñ¿äß';
+    const nameWithSpaces = ' Prompt 1 ';
+    const expectedNameWithSpaces = nameWithSpaces.trim();
+
+    await dialTest.step('Add a dot at the end of a prompt name', async () => {
+      await dialHomePage.openHomePage();
+      await dialHomePage.waitForPageLoaded({
+        isNewConversationVisible: true,
+      });
+      await prompts.openEntityDropdownMenu(prompt.name);
+      await promptDropdownMenu.selectMenuOption(MenuOptions.edit);
+      await promptModalDialog.setField(promptModalDialog.name, newNameWithDot);
+    });
+
+    await dialTest.step(
+      'Check that the name field is red-bordered and an error message appears',
+      async () => {
+        await promptModalAssertion.assertNameFieldIsInvalid();
+      },
+    );
+
+    await dialTest.step(
+      'Fill in the prompt body and click the Save button',
+      async () => {
+        await promptModalDialog.setField(
+          promptModalDialog.prompt,
+          ExpectedConstants.newPromptTitle(1),
+        );
+        await promptModalDialog.saveButton.click();
+      },
+    );
+
+    await dialTest.step('Check that a UI error appears', async () => {
+      await errorToastAssertion.assertToastIsVisible();
+      await errorToastAssertion.assertToastMessage(
+        ExpectedConstants.nameWithDotErrorMessage,
+        ExpectedMessages.notAllowedNameErrorShown,
+      );
+      // Wating for (Closing) the toast to move forward
+      await errorToast.waitForState({ state: 'hidden' });
+    });
+
+    await dialTest.step(
+      'Type restricted characters one by one in the Rename prompt dialog',
+      async () => {
+        for (const char of ExpectedConstants.restrictedNameChars.split('')) {
+          await promptModalDialog.setField(promptModalDialog.name, char);
+          expect
+            .soft(
+              await promptModalDialog.getName(),
+              ExpectedMessages.charactersAreNotDisplayed,
+            )
+            .toBe('');
+        }
+      },
+    );
+
+    await dialTest.step(
+      'Copy and paste restricted characters to the prompt name and verify the name',
+      async () => {
+        await dialHomePage.copyToClipboard(nameWithRestrictedChars);
+        await promptModalDialog.name.click();
+        await dialHomePage.pasteFromClipboard();
+        expect
+          .soft(
+            await promptModalDialog.getName(),
+            ExpectedMessages.promptNameValid,
+          )
+          .toBe(expectedPromptName);
```

Comment:
let's move this assertion to promptModalAssertion class

Reply:
done